app/Providers/AppServiceProvider.php

```diff
@@ -69,9 +73,14 @@ public function boot(): void
 
         $this->app->booted(function () {
             $schedule = $this->app->make(Schedule::class);
+
             $schedule->command(LogUsersOnlineCount::class)->everyThirtyMinutes();
+
             $schedule->command(DeleteExpiredEmailVerificationTokens::class)->daily();
             $schedule->command(DeleteOverdueUserAccounts::class)->daily();
+
+            $schedule->command(CacheMostPopularEmulators::class)->weeklyOn(4, '8:00'); // Thursdays, ~3:00AM US Eastern
+            $schedule->command(CacheMostPopularSystems::class)->weeklyOn(4, '8:30'); // Thursdays, ~3:30AM US Eastern
```

Comment:
Is it sane to only run the command weekly if the cache expires daily?
https://github.com/RetroAchievements/RAWeb/pull/3408/files#diff-2dd96ecf88d62ed5d40d46165162389c8c0cfaa9608e1a0f4243a8e7d38cc20dR59

Reply:
Nope! Looks like I updated the other command, but forgot to update the one you pointed to. Updated in latest.